tests/e2e/scale/noobaa/test_scale_obc_creation_repsin_noobaa_pods.py

```diff
@@ -119,8 +119,8 @@ def test_scale_obc_creation_noobaa_pod_respin(
         )
         log.info(
             f"Number of OBCs in Bound state after node reset: "
-            f"{len(obc_status_list)}"
+            f"{len(obc_status_list[0])}"
```

Comment:
won't this line value would be always 1?

Reply:
obc_status_list contains 2 lists of obc_bound_list and obc_not_bound_list.
len(obc_status_list[0]) will give the total number of obc_bound_list.